Labelbox/labelbox-python
# [PTDT-1108] Added client method unarchive_root_schema_node

How to test:
```
from labelbox import Client

client = Client(
    api_key=<LOCALHOST_API_KEY>,
    rest_endpoint="http://localhost:3000/api/api/v1")

res = client.unarchive_feature_schema_node(<ontology_id>, <feature_schema_id>)
print(res)
```

Given a valid API key that owns the valid ontology and root feature schema node expect `True`


* Important: Test unauthorized access *

Use a valid api key for a user in another org. Attempt to unarchive the root feature schema node belonging to another org. Expect to get a "404" response.

https://www.loom.com/share/379eaebf4f5c483fa0fc0ca6656475d9
